app/controllers/v5/base_systems_controller.rb

```diff
@@ -7,6 +7,7 @@ module BaseSystemsController
     included do
       around_action :with_token
       skip_around_action :with_current_power
+      skip_before_action :check_power, raise: false
```

Comment:
I had to bump consul and add this line otherwise I would get method not found errors.